src/adapters/authAdapter.ts

```diff
@@ -5,19 +5,40 @@ type LoginAuthType = {
   password: string;
 };
 
+/* eslint-disable camelcase */
+export const commonParams = {
```

Comment:
If these are common/shared, shouldn't these be define in the same file as for `BaseAdapter`?

It might be best to rename it to `OauthParams` as it is more descriptive of what these are.